src/orquestra/sdk/_base/cli/_dorq/_ui/_presenters.py

```diff
@@ -210,6 +211,7 @@ def prompt_for_login(self, login_url, url, ce):
         )
 
     def prompt_config_saved(self, url, config_name):
+        # TODO: tell the user what runtime is being used.
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick:** ‏I know you have another PR for this, but please don't leave TODOs without adding a ticket. This time we can probably just remove it.